app.rb

```diff
@@ -0,0 +1,71 @@
+require_relative 'person'
+require_relative 'book'
+require_relative 'rental'
+
+class App
+  def initialize
+    @people = []
+    @books = []
+    @rentals = []
+  end
+
+  def list_books
+    @books.each do |book|
+      puts "Title: #{book.title}, Author: #{book.author}"
+    end
+  end
+
+  def list_people
+    @people.each do |person|
+      puts "Name: #{person.name}, Age: #{person.age}, ID: #{person.id}"
+    end
+  end
+
+  def create_person(type, age, name, parent_permission: true)
+    if type == 'student'
+      classroom = ask_for_classroom
+      person = Student.new(age, classroom, name, parent_permission)
+    elsif type == 'teacher'
+      specialization = ask_for_specialization
+      person = Teacher.new(age, specialization, name, parent_permission)
+    else
+      puts 'Invalid person type.'
+      return
+    end
+    @people << person
+    puts "Person created: Name: #{person.name}, Age: #{person.age}, ID: #{person.id}"
+  end
```

Comment:
- [ ] Hey Ricardo you're doing good so far, but unfortunately, I stumbled across a bug whenever I try to create a student or a person and the issue is coming from the code at lines `26` and `29` `ask_for_classroom` and `ask_for_specialization` aren't defined anywhere in the code and they're causing issues as shown below 👇
  ![image](https://github.com/bohaz/OOP-school-library/assets/104968236/c0866497-74b1-4b96-901d-c3582a0ae088)
  
  ![image](https://github.com/bohaz/OOP-school-library/assets/104968236/196137f6-28fd-4d28-88dc-e78dc94d15c8)

### Note for future reviewers 📝
- [ ] Since a person couldn't be created, rentals weren't also made so please check whether this feature works as expected.